tag_master_node:
  camera_topic_name: "/camera/publisher/color/image"
  camera_info_topic_name: "/camera/publisher/color/camera_info"